snippets/collection-sorting.liquid

```diff
@@ -0,0 +1,36 @@
+<div class="form-horizontal d-flex justify-content-between">
```

Comment:
Can be replaced to the options with absolute URL with sortBy parameter in query string as  values 
```suggestion
<div class="form-horizontal">
  <label for="sortBy" class="small--hide">{{ 'collections.sorting.title' | t }}</label>
  <select name="sortBy" id="sortBy" onchange="location = this.value;">
    <option value="{{ 'priority-descending;name-ascending' | sort_by_url }}">{{ 'collections.sorting.featured' | t }}</option>
    <option value="{{ 'best-selling' | sort_by_url}}">{{ 'collections.sorting.best_selling' | t }}</option>
    <option value="{{ 'title-ascending' | sort_by_url}}">{{ 'collections.sorting.az' | t }}</option>
    <option value="{{ 'title-descending' | sort_by_url}}">{{ 'collections.sorting.za' | t }}</option>
    <option value="{{ 'price-ascending' | sort_by_url}}">{{ 'collections.sorting.price_ascending' | t }}</option>
    <option value="{{ 'price-descending' | sort_by_url}}">{{ 'collections.sorting.price_descending' | t }}</option>
    <option value="{{ 'createddate-descending' | sort_by_url }}">{{ 'collections.sorting.date_descending' | t }}</option>
    <option value="{{ 'createddate-ascending' | sort_by_url }}">{{ 'collections.sorting.date_ascending' | t }}</option>
  </select>
</div>

<script>
  $(function() {
    $('#sortBy')
      // select the current sort order
      .val('{{ current_product_search_criteria.sort_by | default: collection.default_sort_by | sort_by_url }}')
  });
</script>

```